src/main/java/com/adepuu/exercises/session7/Event.java

```diff
@@ -0,0 +1,196 @@
+package com.adepuu.exercises.session7;
+
+import java.util.*;
+
+public class Event {
+  private final List<Ticket> tickets = new ArrayList<>();
+  private final List<Ticket> bookedTickets = new ArrayList<>();
+  private final Scanner scanner = new Scanner(System.in);
+  User user = new User();
+
+  public void getTickets() {
+    try {
+      System.out.println("------List of Tickets-------");
+      if (tickets.isEmpty()) {
+        System.out.println("----- Tickets are empty -----");
+        return;
+      }
+      for (Ticket map : tickets) {
+        System.out.println("Ticket id: " + map.getTicketId());
+        System.out.println("Event name: " + map.getEventName());
+        System.out.println("Ticket name: " + map.getTicketName());
+        System.out.println("Price: " + map.getPrice());
+        System.out.println("--------------------------------");
+      }
+    } catch (Exception e) {
+      System.out.println("Ups Something Error");
+    }
```

Comment:
Try to be more specific about what kind of error we are expecting, This approach can lead to a loss of specific information about the error. OR we can print the actual Exception or at least the stack trace